.nais/naiserator-dev.yaml

```diff
@@ -55,10 +55,9 @@ spec:
   vault:
     enabled: true
     paths:
-      - kvPath: "/serviceuser/data/dev/srvisproxy"
-        mountPath: "/secrets/serviceuser/isproxy"
-      - kvPath: "/kv/preprod/fss/isproxy/default"
-        mountPath: "/secrets/vault"
+      - kvPath: serviceuser/data/dev/srvisproxy
+        mountPath: /secrets/serviceuser/isproxy
```

Comment:
En ting jeg glemte:
Kan droppe Vault helt her egentlig og legge det som en secret i namespacet(siden det er team namespace).
Men det er kanskje ikke så farlig om vi tenker at denne appen sannsynligvis ender opp med å slettes og ikke flyttes GCP.

Reply:
Ja, ser ikke for meg at denne app'en skal flyttes til GCP noen gang - den må vel leve så lenge andre app'er vi er avhengige av ikke kan nås fra GCP. Så da er det neppe noen ulempe å ha Vault-config eksplisitt.